### ISSUE TITLE
mintty 3.7.4 : Failed to run '/usr/bin/bash': Permission denied Exit 126.

### ISSUE THREAD

**Issue by complexconfusion** [git-for-windows/git#5135]
- not able to use git bash on windows any more

### Setup

 - Git for Windows 2.46.0 64-bit

$ git --version --build-options

git version 2.46.0.windows.1
cpu: x86_64
built from commit: 2e6a859ffc0471f60f79c1256f766042b0d5d17d
sizeof-long: 4
sizeof-size_t: 8
shell-path: D:/git-sdk-64-build-installers/usr/bin/sh
feature: fsmonitor--daemon
libcurl: 8.9.0
OpenSSL: OpenSSL 3.2.2 4 Jun 2024
zlib: 1.3.1

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

Microsoft Windows [Version 10.0.19045.4780]

 - What options did you set as part of the installation? Or did you choose the
   defaults?

Editor Option: Notepad++
Custom Editor Path:
Default Branch Option: main
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Disabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Disabled

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

** insert your response here **

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

** insert your response here **

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
clicking on the file explorers right mouse context menu : open git bash here

```
 - What did you expect to occur after running these commands?

opening minttys terminal window showing a user prompt

 - What actually happened instead?

opening minttys terminal window () showing the message :

Failed to run '/usr/bin/bash': Permission denied
/usr/bin/bash: Exit 126.

(with a red background)

**Comment by NicolasReyrolle** [comment]
Same issue here on Windows 10.0.22631, appears that bin/bash.exe is not executable
Was working well on 2.46 until yesterday, and suddenly not possible anymore. I don't know why, but found out this comes from Windows Security, which has blocked the action:
![image](https://github.com/user-attachments/assets/0dc81f56-8548-410c-86b3-5a65e5e65fd7)

**Comment by NicolasReyrolle** [comment]
I investigated a little bit further, and turns out a standard installation works. Mine uses scoop, so I believe this is more related to scoop (and windows policies) than to git-bash

**Comment by dscho** [comment]
Hmm. So it looks like an anti-malware issue? I'll add that label, then.

**Comment by NicolasReyrolle** [comment]
Yes, it looks like so, some default strategy added recently by Microsoft, we believe. That's more on scoop side, I think, but I do not understand why are the binaries are identical

**Comment by dscho** [comment]
Strange, indeed. Let's focus on https://github.com/ScoopInstaller/Main/issues/6190 by closing this here ticket.
